repotracker/repotracker_test.go

```diff
@@ -761,9 +756,16 @@ func TestBuildBreakSubscriptions(t *testing.T) {
 		},
 	}
 	assert.NoError(u4.Insert())
+	assert.NoError(AddBuildBreakSubscriptions(&v1, &proj2))
+	assert.NoError(db.FindAllQ(event.SubscriptionsCollection, db.Q{}, &subs))
+	assert.Len(subs, 2)
```

Comment:
Seems like this should check that the scription for `u2` is now `@hello.itsme` to verify the PR change.

Reply:
That would be ideal but I don't want to add logic for dealing with the Target interface{}. I stepped through it with a debugger though so I know that it's acting the way it should. 
The other tests are not that robust so I don't think it's crucial to add it here.

Reply:
The point of the test is not just to verify that the code work today, but to also verify non-regressive behavior. Just spot checking is not a replacement for a test, which will verify on every PR that the resulting subscription is still correct. You should be able to compare against an `interface{}` type by using `assert.EqualValues`.